client/src/pages/UpdateTaskPage.tsx

```diff
@@ -99,24 +84,14 @@ export const UpdateTaskPage: React.FC<RouteComponentProps<IUpdateMatchParams>> =
               </IonLabel>
             </IonCardContent>
           </IonCard>
-          <form onSubmit={submit} style={{ padding: '0 16px' }}>
-            <IonItem>
-              <IonLabel color="primary" position="floating">Title</IonLabel>
-              <IonInput type="text" name="title" onInput={(e: any) => setTitle(e.target.value)} value={task.title} />
-            </IonItem>
-            <IonItem>
-              <IonLabel color="primary" position="floating">Description</IonLabel>
-              <IonInput type="text" name="description" onInput={(e: any) => setDescription(e.target.value)} value={task.description} />
-            </IonItem>
-            <IonButton className="submit-btn" expand="block" type="submit">Update</IonButton>
-          </form>
+          <TaskForm handleSubmit={submit} model={task} />
           <IonToast
             isOpen={showToast}
             onDidDismiss={() => setShowToast(false)}
             message={errorMessage}
             position="top"
             color="danger"
-            duration={2000}
+            duration={20000}
```

Comment:
It might be too long. Not opinion

Reply:
Oh whoops, that was me debugging. Will fix